bin/adhoc-scripts/mongoInit.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Script to connect to a MongoDB instance based on a set of parameters given at runtime
+Usage:
+      source /data/srv/current/apps/reqmgr2ms/etc/profile.d/init.sh
+      python3 -i -- mongoInit.py -c $WMCORE_SERVICE_CONFIG/reqmgr2ms-output/config-output.py
+"""
+
+import sys
+import logging
+import argparse
+
+from WMCore.Database.MongoDB import MongoDB
+
+from WMCore.Configuration import loadConfigurationFile
+
+if __name__ == '__main__':
+
+    FORMAT = "%(asctime)s:%(levelname)s:%(module)s:%(funcName)s(): %(message)s"
+    logging.basicConfig(stream=sys.stdout, format=FORMAT, level=logging.INFO)
+    logger = logging.getLogger(__name__)
+
+    parser = argparse.ArgumentParser(
+        prog='MongoInit',
+        formatter_class=argparse.RawTextHelpFormatter,
+        description=__doc__)
+
+    parser.add_argument('-c', '--config', required=True,
+                        help="""\
+                        The path to MSCONFIG to be used, e.g.
+                        for production: /data/srv/current/config/reqmgr2ms/config-output.py
+                        """)
+    args = parser.parse_args()
+
+    logger.info("Loading configFile: %s", args.config)
+    config = loadConfigurationFile(args.config)
+
+    msConfig = config.section_('views').section_('data').dictionary_()
+
+    msConfig.setdefault("mongoDBUrl", 'mongodb://localhost')
+    msConfig.setdefault("mongoDBPort", 27017)
```

Comment:
Same answer as bellow. But I agree on one thing, the default database name is indeed badly chosen. I overlooked that because the script was originally developed for another PR related to another micorservice. I'll change that one.